Gatling tests fail to run with Gradle

When running `./gradlew gatlingRun` with the 3.11.0 release, Gatling tests fail to run.

```
* What went wrong:
Execution failed for task ':gatlingRun'.
> Could not get unknown property 'gatlingSimulationClassName' for task ':gatlingRun' of type org.gradle.api.DefaultTask.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

Changing line 31 from:

```
createGatlingRunTask(gatlingSimulationClassName).dependsOn('manifestJar').execute()
```

to:

```
createGatlingRunTask(gatlingSimulationClass).dependsOn('manifestJar').execute()
```

fixes the problem.

--- a/generators/server/templates/_pom.xml
+++ b/generators/server/templates/_pom.xml
@@ -354,6 +354,11 @@
             <version>${gatling.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jzlib</artifactId>
+            <version>1.1.2</version>
+        </dependency>
         <%_ if (databaseType === 'cassandra') { _%>
         <!-- include netty handler explicitly to overcome dependency mismatch when using cassandra -->
         <dependency>

--- a/generators/server/templates/gradle/_gatling.gradle
+++ b/generators/server/templates/gradle/_gatling.gradle
@@ -28,7 +28,7 @@ task gatlingRun(dependsOn:'manifestJar') << {
         gatlingSimulationClass = project.property('gatlingSimulationClass')
     }
 
-    createGatlingRunTask(gatlingSimulationClassName).dependsOn('manifestJar').execute()
+    createGatlingRunTask(gatlingSimulationClass).dependsOn('manifestJar').execute()
 }
 
 task gatlingRunAll(dependsOn: 'manifestJar') << {

